core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -3092,6 +3165,7 @@ private void o3MoveUncommitted0(
             final long columnTop = columnTops.getQuick(colIndex);
 
             if (columnTop > 0) {
+                //noinspection SuspiciousNameCombination
```

Comment:
minor: i disabled the entire inspection pattern. They are aimed to help UI developers

Reply:
ok